Pandator/Assets/Rabbit/Scripts/RabbitMove.cs

```diff
@@ -71,10 +87,30 @@ private void Update()
             forwardDirection.Normalize();
 
             // 移動処理
-            if (!isAButtonPressed){
+            if (!isAButtonPressed && !isKeybord)
+            {
                 transform.Translate(forwardDirection * totalSpeed * Time.deltaTime, Space.World);
-            }else{
-                transform.Translate( JUMP_MOVE_SPEED * forwardDirection * totalSpeed * Time.deltaTime, Space.World);
+            }else if (isKeybord)
+            {
```

Comment:
このままだと
Abuttonを押しているときは動かなくなるからダメです
動きすぎるのもダメなので，JUMP_MOVE_SPEEDをかけてる感じです
```suggestion
            }else if(isAbuttonPressrd && !isKeybord)
            {
                transform.Translate( JUMP_MOVE_SPEED * forwardDirection * totalSpeed * Time.deltaTime, Space.World);
            }else if (isKeybord)
            {
```